101-webapp-for-containers/scripts/deploy.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Generate a unique 12 character alphanumeric string to ensure unique resource names
+uniqueId=$(env LC_CTYPE=C tr -dc 'a-z0-9' < /dev/urandom | fold -w 12 | head -n 1)
+
+# Create a Service Principal
+servicePrincipalName="http://sp-$uniqueId"
+password=$(az ad sp create-for-rbac --name $servicePrincipalName --skip-assignment --query password --output tsv)
+appId=$(az ad sp show --id $servicePrincipalName --query appId --output tsv)
+servicePrincipalObjectId=$(az ad sp show --id $appId --query objectId --output tsv)
+
```

Comment:
remove this file, or if you want to keep if for svc principal creation, remove everything after line 11

Reply:
The Bash script is provided as an example of how to deploy the template including the prereqs outside the context of this repository. It's not meant to be used in the validation of the template itself. 

Should I perhaps the rename the folder to something else than `scripts`?

Reply:
All templates in the repo are deployed the same inside and outside of the repo - having every sample provide their own deployment script creates confusion and a maintenance problem...

Reply:
I removed the script and added instructions for how to deploy the resources in the README file. The build fails for some reason?